group-generators/generators/sismo-community/index.ts

```diff
@@ -0,0 +1,74 @@
+import { dataOperators } from "@group-generators/helpers/data-operators";
+import {
+  GroupStore,
+  GroupWithData,
+  Tags,
+  ValueType,
+} from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Daily,
+  dependsOn: [
+    "sismo-contributors",
+    "sismo-lens-followers",
+    "sismo-zk-badges-holders",
+    "sismo-builders",
+    "sismo-citadel-members",
+    "sismo-core-team",
+  ],
+
+  generate: async (
+    context: GenerationContext,
+    groupStore: GroupStore
+  ): Promise<GroupWithData[]> => {
+
+    const sismoContributors = await groupStore.latest("sismo-contributors");
+    
+    const sismoLensFollowers = await groupStore.latest("sismo-lens-followers");
+    const sismoZkBadgesHolders = await groupStore.latest("sismo-zk-badges-holders");
+
+    const sismoBuilders = await groupStore.latest("sismo-builders");
+
+    const sismoCitadelMembers = await groupStore.latest("sismo-citadel-members");
+    const sismoCoreTeam = await groupStore.latest("sismo-core-team");
+    
+    // Level attributions
+    const level1 = dataOperators.Map(dataOperators.Union([
+      await sismoLensFollowers.data(),
+      await sismoZkBadgesHolders.data(),
+      dataOperators.Selection(await sismoContributors.data(), 1),
+    ]), 1);
+
+    const level2 = dataOperators.Map(dataOperators.Union([
+      await sismoBuilders.data(),
+      dataOperators.Selection(await sismoContributors.data(), 2),
+      dataOperators.Selection(await sismoContributors.data(), 3),
```

Comment:
What is "Selection"? It should be "Filter" with the argument as being { value: 3 }

Reply:
Selection is a new Data Operator I created to have only the account of a certain value of a group.
Bigq merged it yesterday, but yes happy to rename it to filter! 👍
I also think afterwards that we could setup a "EQ", "GTE" etc... (but not a priority at all)

Reply:
Yes it will be more clear